scripts/EN/strtab_script

```diff
@@ -1,849 +1,849 @@
+675: ¥"The target is its abdomen¥"
+677: ¥"Harry says ¥"I can't¥"¥"
+678: ¥"The target is its wings¥"
+679: ¥"The target is its head¥"
+680: ¥"Hiding behind the water tower¥"
+681: ¥"Sounds of a struggle¥"
+682: ¥"Kaufmann is attacked¥"
+683: ¥"After the battle¥"
+684: ¥"The question that had been on my mind¥"
+685: ¥"About Alessa¥"
+686: ¥"Kaufmann's secret¥"
+687: ¥"Garage¥"
+688: ¥"The man who's face turned white with anger¥"
+689: ¥"Showing Kaufmann the picture of Alessa¥"
+690: ¥"On the small boat¥"
+691: ¥"Despair¥"
+692: ¥"A familiar smile¥"
+693: ¥"Dahlia's warning¥"
+694: ¥"Cybil and Harry split up¥"
+695: ¥"Harry heads to the lighthouse¥"
+696: ¥"Harry heads to the amusement park¥"
+697: ¥"Facing Alessa¥"
+698: ¥"The crest on the floor¥"
+699: ¥"The girl disappeared¥"
+700: ¥"Giving up and returning to the boat¥"
+702: ¥"Just where did that girl go?¥"
+703: ¥"The illusion reflected in the spatial distortion¥"
+704: ¥"Jumping into the distortion¥"
+705: ¥"Harry senses danger and turns back¥"
+707: ¥"Rescue of transformed Cybil¥"
+708: ¥"Cybil, from behind¥"
+709: ¥"Plastic Bottle¥"
+710: ¥"Alessa appears in the darkness¥"
+712: ¥"Lisa's whispering¥"
+713: ¥"Responding to Lisa's request¥"
+714: ¥"Harry says ¥"I cannot stay with you¥"¥"
+715: ¥"Lisa's end¥"
+716: ¥"Harry says ¥"Don't worry¥"¥"
+717: ¥"Heading toward the final door¥"
+719: ¥"The rapidly descending room¥"
+720: ¥"Someone is talking…¥"
+721: ¥"Classroom¥"
+722: ¥"Magnetic tape¥"
+723: ¥"The fading illusions¥"
+724: ¥"Child's room¥"
+725: ¥"To freedom¥"
+726: ¥"Firing at transformed Cybil¥"
+727: ¥"Darkness and Cybil¥"
+728: ¥"Harry shoots Cybil¥"
+729: ¥"The appearance of Alessa¥"
+744: ¥"Escape¥"
+746: ¥"Lisa's question¥"
+747: ¥"Talking with Lisa¥"
+748: ¥"The rest of the story¥"
+749: ¥"Kaufmann's suicide¥"
+750: ¥"About Kaufmann¥"
+751: ¥"Switched the bodies¥"
+752: ¥"Heading to the storehouse in the basement with Lisa¥"
+753: ¥"Metal Door¥"
+754: ¥"Complete darkness¥"
+755: ¥"What did you just say…¥"
+756: ¥"Lisa, who's attitude had changed suddenly¥"
+757: ¥"Who are ¥"they¥"?¥"
+758: ¥"Lisa's enlightenment¥"
+759: ¥"Lisa shakes in amazement¥"
+760: ¥"Something is here¥"
+761: ¥"The mystery in the darkness¥"
+762: ¥"What is the sound?¥"
+763: ¥"The crest marked on the door¥"
+764: ¥"The tight cube¥"
+765: ¥"Inside the cramped cube¥"
+766: ¥"Harry opens the door¥"
+767: ¥"Changing the subject¥"
+768: ¥"Mistook him for someone else?¥"
+769: ¥"Show her Alessa's diary¥"
+770: ¥"Getting away is best¥"
+771: ¥"Let my body go with the flow¥"
+772: ¥"The completed manuscript¥"
+773: ¥"Cheryl pouts¥"
+774: ¥"The noisy ringing phone¥"
+775: ¥"Leave it to Cheryl¥"
+776: ¥"Cheryl answers the phone¥"
+777: ¥"Harry goes back to the phone¥"
+778: ¥"Cybil assimilates into the shadows¥"
+779: ¥"A mess¥"
+780: ¥"Feeling of emptiness¥"
+781: ¥"Swaying field of vision¥"
+782: ¥"Frozen body¥"
+783: ¥"Only three bullets remain¥"
+784: ¥"Dazzling light¥"
+785: ¥"Cybil's corpse¥"
```

Comment:
So this is literally "Cybil has become a corpse", but after talking to IlDucci we went with "Cybil's corpse."

"Cybil's body" would work as well, but corpse is more in line with the Japanese and let's the reader know without a doubt that she is dead.